### DISCUSSION TITLE
Ability to Disable YouTube Shorts?

### DISCUSSION THREAD

**Discussion by VanillaEssence** [qnblackcat/uYouPlus#102] [Ideas]
Is it possible to disable YT shorts? I hate them but find myself losing lots of time to them. It would be fantastic to be able to disable them so there would be no temptation to watch.

**Comment by qnblackcat** [comment]
Go to uYou's settings > Other settings and disable YouTube Shorts from there

**Comment by UltraJew** [comment]
Could you please add a option to fully remove shorts? As in them not being present the menu bar, would free up space for the uYou tab as well.
